src/components/TextArea/TextArea.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from '../../packages/bem';
+import styles from './TextArea.scss';
+import { CONTEXTS, SIZES } from '../../constants';
+
+const { block } = bem({
+    name: 'TextArea',
+    classnames: styles,
+    propsToMods: ['context', 'isBlock', 'size']
+});
+
+const TextArea = props => {
+    const { context, disabled, isBlock, size, ...rest } = props;
```

Comment:
did this pass linting? `context`, `isBlock` and `size` are never used. So no need to define them. Or is there?